applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -32,6 +36,29 @@ def _CreateSolver(self):
                 err_msg = "\'run_hrom\' and \'train_hrom\' are both \'true\'. Select either training or running (if training has been already done)."
                 raise Exception(err_msg)
 
+            # Petrov Galerking Training settings
+            self.train_petrov_galerkin = self.rom_parameters["train_petrov_galerkin"]["train"].GetBool() if self.rom_parameters.Has("train_petrov_galerkin") else False
+            if self.train_hrom and self.train_petrov_galerkin:
+                err_msg = "\'train_petrov_galerkin\' and \'train_hrom\' are both \'true\'. Select only one training strategy."
+                raise Exception(err_msg)
```

Comment:
This means that we should most probably condensate these two in a unique input parameter.